test-requirements.txt

```diff
@@ -0,0 +1,16 @@
+# This file is managed centrally.  If you find the need to modify this as a
```

Comment:
Please source this from [the global test requirements](https://github.com/openstack-charmers/release-tools/blob/master/global/source-zaza/test-requirements.txt)